tools/md5sum.wdl

```diff
@@ -1,10 +1,11 @@
-## Description:
-##
-## This WDL file wraps the md5sum tool from the [GNU core utilities](https://github.com/coreutils/coreutils).
-## md5sum is a utility for generating and verifying MD5 hashes.
-
+## [Homepage](https://github.com/coreutils/coreutils)
+#
+# SPDX-License-Identifier: MIT
+# Copyright St. Jude Children's Research Hospital
 version 1.1
 
+# TODO why isn't this in util?
```

Comment:
I actually prefer `md5sum` as a separate file. I think if a task is a straightforward invocation of a tool, then it should go in a per-tool WDL file. Otherwise `util` will become quite unwieldy.

Reply:
In my mind, `util` is a place for utility tasks that are more complex than a single command invocation. I don't know that we've exactly followed that and there is some room for flexibility, but that's how I tend to think about it.

Reply:
IMO, as a `coreutil`, `md5sum` belongs in `util`. Or maybe `md5sum.wdl` could be renamed `coreutils.wdl`? Group it with our `wget` implementation? Is there anything else that would go in there?
However, I'm not going to fight over it, because I think `util` is _already_ unwieldy enough. So preventing another addition to that file is ok by me, even if I don't really agree with the reasoning for keeping it separate.

I would like to explore ways to break up some of our larger files, `util` being among them. I often find it difficult to find the specific task I'm looking for in our long files. Just something I'd like on the back burner, I actually think I'm going to open an Issue for it so we can discuss. I won't list out my full rationale here, but I think there should be at least a soft-cap, if not a hard-cap, on the number of tasks in a WDL file. Don't have an exact number in mind though.

We can continue this conversation when I log that Issue. For now, let's remove this TODO and leave `md5sum.wdl` as is. Unless you like the `coreutils.wdl` rename?

Reply:
I'm fine with deferring a decision for now.